src/test/java/io/candydoc/ddd/extract_ddd_concepts/ExtractDDDConceptsUseCaseTest.java

```diff
@@ -94,8 +98,10 @@ void package_to_scan_is_not_following_ddd() {
 
     // then
     Assertions.assertThatThrownBy(() -> extractDDDConceptsUseCase.execute(command))
-        .isInstanceOf(NoBoundedContextFound.class)
-        .hasMessage("No bounded context has been found in the package : 'wrong.package.to.scan'.");
+        .isInstanceOf(NoBoundedContextNorSharedKernelFound.class)
```

Comment:
rename the test to "no_bounded_context_nor_shared_kernel_in_the_package_to_scan"